src/simulation/deviceInterface/prescribedLinearTranslation/_UnitTest/test_prescribedLinearTranslation.py

```diff
@@ -71,7 +71,7 @@ def test_prescribedLinearTranslation(show_plots,
 
     Args:
         show_plots (bool): Variable for choosing whether plots should be displayed
-        coastOptionRampDuration: (double): [s] Ramp duration used for the coast option
+        coastOptionBangDuration: (double): [s] Ramp duration used for the coast option
```

Comment:
Should the description be "Bang duration", not "Ramp duration"?

Reply:
I had not commited the updated documentation to this draft PR yet, it is committed now.